daemonize.go

```diff
@@ -200,36 +195,29 @@ func Run(
 	}
 }
 
-// Start the daemon process, handing it the supplied pipe for communication. Do
+// Start the daemon process and do
 // not wait for it to return.
-func startProcess(
-	path string,
-	args []string,
-	env []string,
-	pipeW *os.File) (err error) {
+func startProcess(args []string, status io.Writer) error {
+
+	// TODO: drop osext dependency once minmum required go verison is 1.8
```

Comment:
Please just do this now. Those binaries that need to build with older versions of Go can vendor an old version.